python/ray/serve/application.py

```diff
@@ -57,6 +62,28 @@ def __init__(self, deployments: List[Deployment]):
     def deployments(self) -> ImmutableDeploymentDict:
         return self._deployments
 
+    @property
+    def deploy_args_list(self) -> List[Dict]:
```

Comment:
Nice! In the future, we should make this `deploy_args_list` data structure a `dataclass`.